CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ option(CLI_OVPN3 "Build ${PROJECT_NAME} with OpenVPN3 support" OFF)
 cmake_minimum_required(VERSION 3.10)
 
 if(NOT VCPKG_MANIFEST_DIR)
-    set(VCPKG_MANIFEST_DIR ${CMAKE_SOURCE_DIR}/vcpkg_manifests/openssl_3)
+    set(VCPKG_MANIFEST_DIR ${CMAKE_SOURCE_DIR}/vcpkg_manifests)
```

Comment:
Since we do not need two separate manifests, no need put the only manifest into custom directory. 

 - move `vcpkg.json` file to the root source directory, where `vcpkg` searches for it by default
 - remove `vcpkg_manifests` directory
 - remove these lines from `CMakeLists.txt`